src/connector/src/connector_common/common.rs

```diff
@@ -161,21 +162,11 @@ impl AwsAuthProps {
 }
 
 #[serde_as]
-#[derive(Debug, Clone, Deserialize, WithOptions)]
-pub struct KafkaCommon {
+#[derive(Debug, Clone, Deserialize, WithOptions, PartialEq, Hash, Eq)]
+pub struct KafkaConnection {
```

Comment:
It seems we will be able to use this same struct for `CREATE CONNECTION` later? 🤔

Reply:
yes.